src/flow/darcy_flow_mh.cc

```diff
@@ -104,7 +104,7 @@ const it::Selection & DarcyFlowMH::EqData::get_bc_type_selection() {
                        "Specify the pressure head through the 'bc_pressure' field "
                        "or the piezometric head through the 'bc_piezo_head' field.")
                .add_value(neumann, "neumann", "Neumann boundary condition. Prescribe water outflow by the 'bc_flux' field.")
-               .add_value(robin, "robin", "Robin boundary condition. Water outflow equal to $\\sigma (h - h^R)$. "
+               .add_value(robin, "robin", "Robin boundary condition. Water outflow equal to (($sigma (h - h^R)$)). "
```

Comment:
need back slash before sigma:
valid Latex formula is:
\sigma(h-h^R)
how to deal with it? I suggest writing single backslash here and do correct escaping in JSONmachine output if it is not done yet.